ItemList/Source/ItemList.Database/Repositories/ItemsRepository.cs

```diff
@@ -12,12 +12,12 @@ public class ItemsRepository : IItemsRepository
     {
         private readonly IMongoCollection<Item> _collection;
         private const string CollectionName = "Items";
-        private const string DatabaseName = "itemlist";
 
         public ItemsRepository(IDatabaseConfiguration databaseConfiguration)
         {
+            var databaseName = MongoUrl.Create(databaseConfiguration.DefaultConnectionString).DatabaseName;
```

Comment:
You can actually store the `mongoUrl` and use it the `MongoClient` constructor (and then use its `DatabaseName` property).